### PR TITLE
CODE RUB: Upgrade Test to Standard `2.10.1`

### PR DESCRIPTION
**Pull request by aosunlana**
Closes: #102

### CODE REVIEW

File: ADotNet/Services/Builds/BuildService.Exceptions.cs [hunk 1]
```diff
@@ -23,49 +24,75 @@ private static void TryCatch(ReturningNothingFunction returningNothingFunction)
...
             {
-                throw new BuildServiceException(exception);
+                throw CreateBuildServiceException(exception);
             }
         }
+
+        private static AdoValidationException CreateAdoValidationException(Xeption innerException)
+        {
+            return new AdoValidationException(
+                "Ado validation exception occurred, try again", innerException);
```
Comment (hassanhabib) [inline ADotNet/Services/Builds/BuildService.Exceptions.cs:74]: use parameters names and one parameter per line if not on the same line as the exception initialization

File: ADotNet/Services/Foundations/Files/FileService.Exceptions.cs [hunk 2]
```diff
@@ -22,40 +23,74 @@ private static void TryCatch(ReturningNothingFunction returningNothingFunction)
...
+
+            throw new FileDependencyException(
+                "File dependency error occurred, contact support.",
+                failedFileSerializationException);
+        }
+        
+        private static FileServiceException CreateFileServiceException(
+            Exception innerException)
+        {
+            var failedFileServiceException =
```
Comment (hassanhabib) [inline ADotNet/Services/Foundations/Files/FileService.Exceptions.cs:86]: We may need to initialize this guy _inside_ the `catch` block becuase a Service Exception can be used for multiple other inner exception types. Same thing goes with other categorical exceptions

File: ADotNet/Services/Builds/BuildService.Exceptions.cs (hunk 1 above)
Comment (aosunlana) [reply ADotNet/Services/Builds/BuildService.Exceptions.cs]: Treated

**Comment by glhays** [comment]
Closes #21

**Pull request merged**